helm/ffc-sfi-map-web/templates/config-map.yaml

```diff
@@ -7,4 +7,7 @@ data:
   {{- if .Values.container.googleTagManagerKey }}
   GOOGLE_TAG_MANAGER_KEY: {{ quote .Values.container.googleTagManagerKey }}
   {{- end }}
+  {{- if .Values.container.googleTagManagerKey }}
```

Comment:
References the wrong variable, should be `osMapApiKey` not `googleTagManagerKey`